pkg/workflows/workload/validate.go

```diff
@@ -0,0 +1,59 @@
+package workload
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/task"
+	"github.com/aws/eks-anywhere/pkg/validations"
+)
+
+type setAndValidateWorkloadTask struct{}
+
+// SetAndValidateTask implementation
+
+// Run upgradeCluster performs actions needed to upgrade the workload cluster.
+func (s *setAndValidateWorkloadTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	logger.Info("POC Performing setup and validations")
```

Comment:
now i'm just curious
why the same similar log everywhere? also tbh i'm a bit confused by the term poc
not big deal if you want to keep this while you test but it seems like extra work to clean later